openedx/core/djangoapps/appsembler/api/v2/views.py

```diff
@@ -112,7 +112,7 @@ def create(self, request):
         username_exists_error = username_exists(username=username)
         if email_exists_error and username_exists_error:
             errors = {
-                "user_message": "Both email and username already exist",
+                "title": "Both email and username already exist",
```

Comment:
Please use `user_message` for all the error messages instead of `title`. This is more consistent with the error above `'user_message': '{0} is not a valid value for "send_activation_email"'.format(...` and the v1 API.

```suggestion
                "user_message": "Both email and username already exist",
```